python/revng/cli/revng.py

```diff
@@ -35,6 +35,7 @@ def run_revng_command(arguments, options: Options):
             if (
                 isclass(maybe_command)
                 and maybe_command != Command
+                and not isabstract(maybe_command)
```

Comment:
So, I'd say let's drop magic `Command` inheritors and have a `install`/`setup` function as you were suggesting yesterday.

Reply:
Added